### PR TITLE
add QuickCheck CI

### PR DESCRIPTION
**Pull request by tmayse**

### CODE REVIEW

File: data/tools/quickcheck-ci.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    "tags": [
+      "linux",
+      "windows",
+      "osx",
+      "bsd",
+      "solaris",
+      "commercial",
+      "free (beer)",
+      "Quviq Licence http://quickcheck-ci.com/EQC_CI_LICENCE.txt",
+      "ci",
```
Comment (rdsubhas) [inline data/tools/quickcheck-ci.json]: Build failed here, can you please remove the extra comma...

**Comment by tmayse** [comment]
@rdsubhas this should do it.

**Comment by scottmuc** [comment]
@tmayse thanks for the fixing and contributing!

**Comment by rdsubhas** [comment]
:+1:

**Pull request merged**